src/Appwrite/GraphQL/Schema.php

```diff
@@ -98,14 +98,17 @@ protected static function api(App $utopia, callable $complexity): array
             foreach ($routes as $route) {
                 /** @var Route $route */
 
-                $namespace = $route->getLabel('sdk.namespace', '');
-                $method = $route->getLabel('sdk.method', '');
-                $name = $namespace . \ucfirst($method);
+                /** @var \Appwrite\SDK\Method $sdk  */
+                $sdk = $route->getLabel('sdk', false);
 
-                if (empty($name)) {
+                if (empty($sdk)) {
                     continue;
                 }
 
+                $namespace = $sdk->getNamespace();
+                $method = $sdk->getMethodName();
+                $name = $namespace . \ucfirst($method);
```

Comment:
We'll need to handle multiple methods here as well, if there a multiple SDK methods, there should be multiple GraphQL queries too